system/liquid-ceph/templates/configmap.yaml

```
@@ -23,3 +23,10 @@ data:
 
     [client.admin]
     keyring = /etc/ceph-keyring/keyring
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: liquid-ceph-zone-mapping
+data:
+  zone-mapping.yaml: ""
```

To directly specify the yaml in the values file and render it out here I think this should do it.
```suggestion
  zone-mapping.yaml: | {{- toYaml .Values.ceph.zoneMapping | nindent 4 }}
```